### PR TITLE
Render crops for featured images

### PR DESCRIPTION
**Pull request by vied12**
depends on https://github.com/superdesk/superdesk/pull/1903

* Perform renditions when featured image has been crop
* Add a `authoring/services.js` file for new services. (original file is now too long)

### CODE REVIEW

File: scripts/superdesk-authoring/services.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+            .then(function(result) {
+                var renditionNames = [];
+                var savingImagePromises = [];
+                angular.forEach(result.cropData, function(croppingData, renditionName) {
+                    // if croppingData are defined
+                    if (angular.isDefined(croppingData.CropLeft) && !isNaN(croppingData.CropLeft)) {
+                        renditionNames.push(renditionName);
+                    }
+                });
+                console.log('renditionNames', renditionNames, 'result', result);
```
Comment (akintolga) [inline scripts/superdesk-authoring/services.js]: console.log is left here...

**Comment by akintolga** [comment]
:+1: Looks good. Maybe one simple test for the new service would be useful.

**Comment by mdhaman** [comment]
:+1:

**Comment by vied12** [comment]
@akintolga I won't write a test now because we have to change this workflow in the next task.
Let's do it when it will be done.

**Pull request merged**